### PR TITLE
Added types for koa-views

### PR DESCRIPTION
**Pull request by brooklynDev**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: koa-views/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (mhegazy) [inline koa-views/tsconfig.json]: `true` please.

File: koa-views/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
+// Type definitions for koa-views v2.x
```
Comment (mhegazy) [inline koa-views/index.d.ts]: Please add tslint.json to your project, this should catch this invalid version string. see https://github.com/DefinitelyTyped/DefinitelyTyped#lint for details

**Review by mhegazy — DISMISSED** [review]

**Comment by paulvanbrenk** [comment]
@mhegazy does this look good now? 

@brooklynDev can you fix the merge issues?

**Review by mhegazy — COMMENTED** [review]
The new package seems good, i do not know much about the other changes, so i would defer to other reviewers.

**Comment by andy-ms** [comment]
Something went wrong with git and this includes too many changes. Could you rebase on top of `master` and make sure the branch includes just your changes?

**Comment by andy-ms** [comment]
We've recently updated the repository structure.
All packages are now in a `types` subdirectory.
Please rebase this pull request onto the latest `master` branch.

**Comment by brooklynDev** [comment]
@andy-ms I think I got it right this time, if not let me know. Thanks!

**Pull request merged**